django/cantusdb_project/main_app/admin/institution.py

```diff
@@ -1,7 +1,21 @@
 from django.contrib import admin
+from django.urls import reverse
+from django.utils.safestring import mark_safe
 
 from main_app.admin.base_admin import BaseModelAdmin
-from main_app.models import Institution, InstitutionIdentifier
+from main_app.models import Institution, InstitutionIdentifier, Source
+
+
+class InstitutionSourceInline(admin.TabularInline):
```

Comment:
I'm slightly confused by this, because this suggests that sources should be editable, but the `fields` and `readonly_fields` overlap means they're not. Is this used just because it templates nicely? And is not meant to be editable?